terraform/modules/helm/dataset_api/dataset-api-helm-chart/values.yaml

```diff
@@ -30,10 +39,10 @@ redis_service:
 exhaust_service:
   CLOUD_STORAGE_PROVIDER: "aws"
   CLOUD_STORAGE_REGION: "us-east-2"
-  CONTAINER: "obsrv-sb-dev-165052186109"
```

Comment:
Are we mentioning in the document to define the container name?

Reply:
yes